infra/base-images/all.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash -eux
-# Copyright 2016 Google Inc.
+# Copyright 2025 Google LLC
```

Comment:
We usually don't update copyright year

Reply:
Thanks for pointing that out. I see that it was indeed breaking a CI check. I'm going to verify if this copyright update is necessary for existing files, as it's possible it only applies to new ones.